controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -2794,16 +2794,16 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
         break;
     }
     case beerocks_message::ACTION_CONTROL_CLIENT_DHCP_COMPLETE_NOTIFICATION: {
-        auto notification =
+        auto notification_in =
             beerocks_header
                 ->addClass<beerocks_message::cACTION_CONTROL_CLIENT_DHCP_COMPLETE_NOTIFICATION>();
-        if (notification == nullptr) {
+        if (!notification_in) {
```

Comment:
Should mention these fixes in the commit message (nitpick)

Reply:
I changed the variable name from `notification` to 'notification_in` since I had to add new tlv 'notification_out`. I seems weird to me to mention it.